Fix TracerouteResult.world_trace() and set new defaults for matplotlib graphs

p-l- opened the pull request:
Fix `TracerouteResult.world_trace()` (fixes #28) using Matplotlib and [GeoIP](https://pypi.python.org/pypi/GeoIP/) (instead of the internal geoip sub-module, which has been removed), and set new defaults for Matplotlib graphs (see [updated notebook](https://github.com/p-l-/scapy/blob/fix-geoip/doc/notebooks/graphs-ipids.ipynb)).

guedou commented:
Thanks, I will have a look. We definitely need to share our TODOs. I was rewritten this part myself =)

guedou commented on scapy/layers/inet.py:
```diff
@@ -1139,19 +1141,42 @@ def world_trace(self):
+                if line.startswith('#'):
+                    continue
+                curline.append(line.split())
+            print "PLOT"
```
ooops =)

guedou commented on scapy/layers/inet.py:
```diff
@@ -1139,19 +1141,42 @@ def world_trace(self):
+            for line in fdesc:
+                line = line.strip()
+                if not line:
+                    print "PLOT"
```
ooops =)

guedou commented:
I was working on a similar patch myself. I do agree that the best solution is to remove the legacy Scapy geoip, however I think that the best path is too use http://matplotlib.org/basemap/ in order to definitely remove the dependency on gnuplot and the world.dat file that might be hard to retrieve.

I can provide the additional fix, as my patch uses it.

p-l- commented:
Thanks @guedou for the careful review! I have removed `print` statements, and rebased against current master.

For your additional fix, that would be great! Do you prefer to merge this one first and then propose another PR, or to create a PR against my branch (p-l-/scapy:fix-geoip) which will update this PR once merged? The former is probably easier, but the latter is fine too.

guedou commented:
ACK. I will merge this PR.
